tests/integration-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "voxel51-platform-sdk-integration-test",
+  "version": "1.0.0",
+  "description": "Server and integration test suite to permit analytic developers local testing of images prior to platform upload.",
+  "main": "./server/index.js",
+  "repository": "https://github.com/voxel51/platform-sdk",
+  "author": "Voxel51",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "busboy": "^0.3.1",
+    "joi": "^14.3.1",
+    "ramda": "^0.26.1",
+    "uuid": "^3.3.2"
+  },
+  "scripts": {
+    "start": "node server/index.js",
```

Comment:
Speaking as a user of this, it would be more familiar to me if I were able to run my tests via an executable (e.g., bash script) with commandline args. Like this:
```
bash run_test.bash --analytic-json '<json-path>' --analytic-image '<image-path>' ...
```

Stylistically, environment variables are typically only used for things that one "sets and forgets". I would expect to specify things like analytic JSON/image paths, input files, etc. via commandline flags, because they change for every test that I run.